### PR TITLE
Add user mention for .artisttracks and general info about how to tag users

### PR DESCRIPTION
**Pull request by intcreator**
I noticed this functionality existed but it did not appear to be documented.

### CODE REVIEW

File: docs/commands/index.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 To view a list of all available commands, use the sidebar. The basic commands to get started are listed below.
 
+.fmbot supports identifying users by Discord mention, Discord ID, or Last.fm username.
```
Comment (th0mk) [inline docs/commands/index.md:5]: I'm not sure if we should add this since this only applies to a few commands.

File: docs/commands/artists.md [hunk 2]
```diff
@@ -88,14 +88,16 @@ Options:
 
 ### .artisttracks (`.at`)
 
-Shows you your all-time top tracks for the artist you're currently listening to or searching for.
+Shows you your all-time top tracks for the artist you're currently listening to or searching for. You can also show top tracks for someone else in the server.
```
Comment (th0mk) [inline docs/commands/artists.md:91]: Can you make this in line with how we word this in other commands?

So `Shows you or someone else their all-time top tracks for the artist you're currently listening to or searching for.`

And then a list of possible parameters, similarly to the `.topartists` command.

**Review by th0mk — CHANGES_REQUESTED** [review]

File: docs/commands/index.md (hunk 1 above)
Comment (intcreator) [reply docs/commands/index.md:5]: I know it only applies to a few commands, is there a way we can specify that where identifying a user is supported, it always supports those parameters? it looks like we have standard text for other commands as you mentioned in your other comment, should we just update that standard text to include `Discord mention, Discord ID, or Last.fm username`? that info is extremely useful as a power user who doesn't want to ping other Discord users all the time but I had to ask in the .fmbot Discord before I found out this was the case

**Comment by th0mk** [comment]
I've adjusted the docs so the User option is now more clear. I've also gone through all commands and made sure they all have the option when it's available.

Commit: https://github.com/fmbot-discord/docs/commit/ca986ddd74f5b472109bcfa89d99edd6759ea1cc

Thanks for pointing out this needed to be improved.

**Pull request closed without merging**